projects/Amlogic/README.md

```diff
@@ -0,0 +1,16 @@
+# Amlogic
+
+This project is for Amlogic SoC devices
+
+## Links
+
+* http://linux-meson.com
+
+## Useful debug commands
+
+* `cat /sys/kernel/debug/cec/cec0/status`
```

Comment:
none of these commands are specific to amlogic

Reply:
Is that a bad thing? I found it useful to have that kind of readme for Rockchip, so I copied the file and amended a bit for Amlogic: https://github.com/LibreELEC/LibreELEC.tv/blob/master/projects/Rockchip/README.md

Reply:
I'd rather move non platform specific things to a generic document. Not a big deal at this time.